packaging/tests/test_jar_signer.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from pathlib import Path
+from unittest.mock import patch 
+from package.jar_jdk_signer import jdk_sign_jar
+
+
+TEST_FOLDER = Path("tests/test_resources/test_taco_to_sign")
+TEST_TACO = "postgres_odbc.taco"
+TEST_ALIAS_DIFF_PWD = "key_diff_pwd"
+TEST_ALIAS_SAME_PWD= "key_same_pwd"
+TEST_KEYSTORE = "tests/test_resources/test_keystore/test_ks.jks"
+
+
+class TestJarSigner(unittest.TestCase):
```

Comment:
I _think_ you can apply the `patch` here and then not have to repeat it for each test method. [Here's the python docs about it](https://docs.python.org/3.5/library/unittest.mock-examples.html#applying-the-same-patch-to-every-test-method)

```suggestion
@patch('package.jar_jdk_signer.get_user_pwd')
class TestJarSigner(unittest.TestCase):
```

I haven't tried the above, so be sure to test.